<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow2</class>
 <widget class="QMainWindow" name="MainWindow2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>788</width>
    <height>492</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:white</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>711</width>
      <height>411</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#EEEEEE</string>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QScrollArea" name="scrollArea">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>551</width>
        <height>351</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>549</width>
         <height>349</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton_refreshAiHistory">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>40</y>
        <width>80</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>刷新</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2"/>
   </widget>
   <widget class="QPushButton" name="pushButton_Document">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>DOC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Document_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>AI</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>361</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>10</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_name">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>10</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>name</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>788</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
